mappings/net/minecraft/util/ArraySet.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_4706 net/minecraft/util/ArraySet
+	CLASS class_4707 Iterator
+		FIELD field_21566 start I
+		FIELD field_21567 index I
+	FIELD field_21562 sortingFunction Ljava/util/Comparator;
```

Comment:
`comparator` should suffice.

Reply:
Comparitor is used for sorting, though. :/

Reply:
Agreed with Sollace, I'd rather have a more descriptive name than redundant information.

Reply:
Isn't this already a **sorting** function? Is this like a navigable set?

Reply:
It's technically an `OrderedSet`.

In fact, maybe that's a better name.